stdlib/printexc.mli

```diff
@@ -193,7 +193,7 @@ val backtrace_slots : raw_backtrace -> backtrace_slot array option
     recent function call, raise, or primitive [get_backtrace] call in
     the trace.
 
-    Some possible reasons for returning [None] are as follow:
+    Some possible reasons for returning [None] are as follows:
```

Comment:
I have reverted the change, as it indeed is not as obvious as I thought. For once, it's clearly not a crude grammatical mistake.

If everything else is all right, I hope we can merge now.